testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/messaging/mgmt/JMSBridgeManagementTestCase.java

```diff
@@ -0,0 +1,362 @@
+import org.jboss.as.arquillian.api.ContainerResource;
+import org.jboss.as.arquillian.api.ServerSetupTask;
+import org.jboss.as.arquillian.container.ManagementClient;
+import org.jboss.as.controller.PathAddress;
+import org.jboss.as.controller.PathElement;
+import org.jboss.as.controller.client.ModelControllerClient;
+import org.jboss.as.controller.client.helpers.Operations;
+import org.jboss.as.controller.operations.common.Util;
+import org.jboss.as.test.integration.common.jms.JMSOperationsProvider;
+import org.jboss.as.test.integration.management.base.AbstractMgmtTestBase;
+import org.jboss.as.test.integration.security.common.Utils;
+import org.jboss.as.test.shared.TestSuiteEnvironment;
+import org.jboss.dmr.ModelNode;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * @author baranowb
+ * 
+ */
+@RunWith(Arquillian.class)
+@RunAsClient
+public class JMSBridgeManagementTestCase extends AbstractMgmtTestBase {
+
+    private ManagementClient managementClient;
+    @ArquillianResource
+    private static ContainerController container;
+
+    static final PathAddress ADDRESS_HORNETQ = PathAddress.pathAddress(PathElement.pathElement(SUBSYSTEM, SUBSYSTEM_NAME),
+            PathElement.pathElement(SERVER, "default"));
+    static final PathAddress ADDRESS_XA_CONNECTION_FACTORY = ADDRESS_HORNETQ.append(PathElement.pathElement(CONNECTION_FACTORY,
+            "XAConnectionFactory"));
+    static final PathAddress ADDRESS_QUEUE_SOURCE = ADDRESS_HORNETQ.append(PathElement.pathElement(JMS_QUEUE, "sourceQueue"));
+    static final PathAddress ADDRESS_QUEUE_TARGET = ADDRESS_HORNETQ.append(PathElement.pathElement(JMS_QUEUE, "targetQueue"));
+
+    static final PathAddress ADDRESS_JMS_BRIDGE = PathAddress.pathAddress(PathElement.pathElement(SUBSYSTEM, SUBSYSTEM_NAME),
+            PathElement.pathElement(JMS_BRIDGE, "outBridge"));
+
+    static final PathAddress ADDRESS_SOCKET_BINDING = PathAddress.pathAddress(PathElement.pathElement("socket-binding-group","standard-sockets"),
+            PathElement.pathElement("socket-binding","for-netty"));
+    
+    static final PathAddress ADDRESS_NETTY_CONNECTOR = ADDRESS_HORNETQ.append(PathElement.pathElement("remote-connector", "netty"));
+    static final PathAddress ADDRESS_NETTY_ACCEPTOR = ADDRESS_HORNETQ.append(PathElement.pathElement("remote-acceptor", "netty"));
+
+    public static final String CONTAINER = "jbossas-messaging-ha-server1";
+    public static final String DEPLOYMENT_NAME = "DUMMY";
+    
+    private List<ServerSetupTask> tasks = new ArrayList<ServerSetupTask>(4);
+    private List<ServerSetupTask> tasksToClean = new ArrayList<ServerSetupTask>(4);
+    
+    public JMSBridgeManagementTestCase(){
+        
+        tasks.add(new SetupNetty());
```

Comment:
Because I dont want it to run on server side. ARQ has nasty set of rules that alter execution place. (at least I think this was reason for such design, its been quite a bit since wrote this)